src/dashboards/components/DashboardHeader.tsx

```diff
@@ -206,11 +241,14 @@ const mstp = (state: AppState) => {
   const timeRange = getTimeRange(state)
   const org = getOrg(state)
 
+  const autoRefresh = state.autoRefresh[state.currentDashboard.id]
```

Comment:
Given the dependent logic of this statement, I’d love to see some optional chaining so the app doesn’t fall apart if one of those assertions is false